openpype/modules/deadline/repository/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -59,6 +192,91 @@ def get_openpype_executable():
     return exe_list, dir_list
 
 
+def get_openpype_versions(exe_list, dir_list):
```

Comment:
Removed unused argument
```suggestion
def get_openpype_versions(dir_list):
```